src/resources/views/cart.blade.php

```diff
@@ -0,0 +1,14 @@
+@extends('lab19\cart::app')
```

Comment:
@AlexeiaW for blade views / routes we want a few basic tests to confirm these are functional. You should be able to do a simple test of asserting information in the response. 

I'm assuming testbench will be fine with http tests, let me know if not though: https://laravel.com/docs/6.x/http-tests

Reply:
made simple test for the views